3754/1/libs/check_url.rb

```diff
@@ -0,0 +1,17 @@
+require './libs/check_source_code_url.rb'
+require './libs/check_github.rb'
+# class for checking homepage url
+class CheckURL
+  def initialize(homepage_url, source_code_url)
+    @homepage_url = homepage_url.to_s
+    @source_code_url = source_code_url.to_s
+  end
+
+  def check_url
+    check_source_url = CheckSourceCodeURL.new(@source_code_url)
+    check_git_for_hp = CheckGithub.new(@homepage_url)
```

Comment:
you check twice for github, is there any reason to not trust to the first check? :)